SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialSender.cpp

```diff
@@ -470,7 +411,7 @@ void USpatialSender::SendComponentUpdates(UObject* Object, const FClassInfo& Inf
 	for (int i = 0; i < ComponentUpdates.Num(); i++)
 	{
 		FWorkerComponentUpdate& Update = ComponentUpdates[i];
-		if (!NetDriver->StaticComponentView->HasAuthority(EntityId, Update.component_id))
+		if (!NetDriver->StaticComponentView->HasAuthority(EntityId, SpatialConstants::SERVER_AUTH_COMPONENT_SET_ID))
```

Comment:
Can use `SpatialNetDriver::HasServerAuthority` here, and just run it once before iterating over the components. This looks like more of a critical error now.